### DISCUSSION TITLE
change user agent in request

### DISCUSSION THREAD

**Discussion by WalidMahmoud101** [FlareSolverr/FlareSolverr#751] [Q&A]
how i can change user agent in req post data on the program?

**Comment by ngosang** [comment]
You can't change the User-Agent. It will be detected by Cloudflare.

**Reply by WalidMahmoud101** [reply]
Bro how can I use a proxy in the script I can't When I set up a proxy and make a request to http://httpbin.org/ip for Check IP 

It shows my IP, not the proxy IP :
https://i.imgur.com/brnjcwy.png

Even though I put the correct Postdata

**Reply by ngosang** [reply]
Proxy param has been implemented in the latest version => https://github.com/FlareSolverr/FlareSolverr/releases/tag/v3.2.0

**Reply by ming-jin-dev** [reply]
How can I use a proxy to initiate or make requests? Can you provide me with an example?

**Reply by WalidMahmoud101** [reply]
Url Req : http://localhost:8191/v1
postData :
{"cmd":"request.get","url":"","maxTimeout":5000,"proxy": {"url": "ip:port"}}

**Reply by ming-jin-dev** [reply]
Thank you very much. Good people have a safe life~
